assignments/callbacks.js

```diff
@@ -1,34 +1,68 @@
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
+function callBack(output, bool) {
```

Comment:
Why does your callBack function take 2 arguments? It may only need 1 from what I see in the code.
`function callBack(output) { ....`